streamalert_cli/terraform/common.py

```diff
@@ -25,6 +25,10 @@ class InvalidClusterName(Exception):
     """Exception for invalid cluster names"""
 
 
+class MisconfigurationError(ValueError):
```

Comment:
Nice 👍

Reply:
Maybe put this into `streamalert.shared` or `streamalet.shared.exceptions` or something